test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -443,7 +435,7 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 				gomega.Eventually(func(g gomega.Gomega) {
 					g.Expect(k8sClient.Get(ctx, wlKey, &updatedWl)).To(gomega.Succeed())
 					g.Expect(workload.IsEvictedByDeactivation(&updatedWl)).To(gomega.BeTrue())
-					util.ExpectEvictedWorkloadsTotalMetric(cq.Name, kueue.WorkloadEvictedByDeactivation, 1)
+					util.ExpectEvictedWorkloadsTotalMetric(cq.Name, kueue.WorkloadEvictedByDeactivation+kueue.WorkloadEvictedByAdmissionCheck, 1)
```

Comment:
Is the metric `kueue.WorkloadEvictedByAdmissionCheck` increased in this scenario?

Reply:
It is not since:
```
workload.ReportEvictedWorkload(r.recorder, wl, cqName, kueue.WorkloadEvictedByAdmissionCheck, message)
```
was not called

Reply:
Then could we discard the changes where the metric was not increased?

Reply:
@PBundyra do you mean to revert the removal of the "ginkgo.By("Checking the admission check", func() {" above, or something else?

Reply:
The scenario looks good to me, I only meant the change about checking the metric but I see now that it's because of reason concatenation